l10n_it_fatturapa/demo/account_invoice_fatturapa.xml

```diff
@@ -17,12 +18,13 @@
                 (0,0, {
                     'factor_percent': 100,
                     'repartition_type': 'tax',
-                    'account_id': ref('l10n_generic_coa.tax_received'),
+                    'account_id': ref('l10n_it.2601', raise_if_not_found=False) or ref('l10n_generic_coa.tax_payable', raise_if_not_found=False) or obj().env['account.account'].search([('user_type_id', '=', ref('account.data_account_type_current_liabilities'))], limit=1),
```

Comment:
In questo e negli altri `search`, ho notato che manca un `.id` in fondo, lo potresti aggiungere?
Se ricordo bene quando si modifica un campo m2o in questo modo, il valore deve essere un intero o False (analogamente al risultato di `ref`).

Reply:
Intendi così?